test/NuGet.Server.Core.Tests/ServerPackageRepositoryTest.cs

```diff
@@ -356,6 +356,139 @@ public class ServerPackageRepositoryTest
             }
         }
 
+        [Fact]
+        public async Task ServerPackageRepositorySearchDelistingDisabledAndExclude()
```

Comment:
These tests are very similar, you could likely reduce them with a helper function that takes in
- whether `EnableDelisting` is provided
- whether or not the package is returned by search
- whether or not the search is performed with `includeDelisted`
- whether or not the package is returned in the search

Reply:
Thanks, done.